vyper/opcodes.py

```diff
@@ -30,7 +30,7 @@
     'BYTE': [0x1a, 2, 1, 3],
     'SHA3': [0x20, 2, 1, 30],
     'ADDRESS': [0x30, 0, 1, 2],
-    'BALANCE': [0x31, 1, 1, 400],
+    'BALANCE': [0x31, 1, 1, 700],
```

Comment:
Can you add a comment to the effect of ISTANBUL RULESET

Reply:
```suggestion
    'BALANCE': [0x31, 1, 1, 700],  # Under Istanbul Ruleset
```